Makefile

```diff
@@ -20,7 +21,12 @@ update-license:
 
 build-dars: lib-damldir certificates-damldir finance-damldir banking-damldir demoadmin-damldir landlord-damldir testing-damldir triggers-damldir reset-damldir
 
-include scripts/dependencies.mk
+# re-builds and re-reads the dependency graph as needed
+# (see https://www.gnu.org/software/make/manual/html_node/Remaking-Makefiles.html)
+DAR_DEPENDENCY_GRAPH=dependencies.mk
+include $(DAR_DEPENDENCY_GRAPH)
+$(DAR_DEPENDENCY_GRAPH): $(shell find -name daml.yaml)
```

Comment:
was this makeDeps.sh already there?

Reply:
because we used it to manually update `scripts/dependencies.mk` (removed in this PR, pls look into the deleted content)